src/sidebar/services/oauth-auth.js

```diff
@@ -1,10 +1,12 @@
-import EventEmitter from 'tiny-emitter';
+import { TinyEmitter } from 'tiny-emitter';
```

Comment:
As in another recent PR, `TinyEmitter` is exactly the same as `EventEmitter`, but only `TinyEmitter` has type information available in the `tiny-emitter` package (without us having to write our own type info for that package).